osc/commands_git/pr_create.py

```diff
@@ -17,20 +18,35 @@ def get_editor() -> str:
         candidates = ["vim", "vi"]
 
     editor_path = None
+    args = None
     for i in candidates:
-        editor_path = shutil.which(i)
+        i, *args = i.split()
```

Comment:
I think `shlex.split()` makes more sense here, because theoretically you can have `EDITOR=foo --option="value with spaces"`.